docs/topics/proc_deploying-the-secured-booster-using-the-oc-cli-client.adoc

```diff
@@ -5,11 +5,12 @@
 // Rationale: This procedure is identical in all deployments
 
 [id='deploying-the-secured-booster-using-the-oc-cli-client_{context}']
-= Deploying the {name-mission-secured} booster using the `oc` CLI client
+[id='deploying-the-secured-booster-using-the-oc-client_{context}']
+= Deploying the {name-mission-secured} example using the `oc` CLI client
```

Comment:
Deploying the {name-mission-secured} example -->  Deploying the {name-mission-secured} example application